index.html

```diff
@@ -8,34 +8,42 @@
     </head>
     <body>
         <header class="header">
-            <span class="logo-link">
+            <div class="logo-link">
```

Comment:
by the way, on the main page logo doesn't lead anywhere

Reply:
Agree, done